study.md

```diff
@@ -1,4 +1,4 @@
-# Single-Page Applications Study
+## Single-Page Applications Study
 
```

Comment:
Good catch, but feel free to open issues for markdown corrections, typos or anything like that you see.  I'll open on for this.